app/assets/stylesheets/shareable.scss

```diff
@@ -0,0 +1,10 @@
+nav.share {display:inline-block;} /* shareable's nav tag */
+nav.share div {float:left;}
+nav.share iframe {border:0; margin-top:0; padding-top:0;} /* if using medium sized buttons */
+#___plusone_0 {font-size: default !important;}
+.fb_iframe_widget span { vertical-align: top; }
+.fb_edge_widget_with_comment { vertical-align: top; }
+.fb-like{display:inline-block}
+* html .fb-like{display:inline}/* ie6 inline block fix*/
+*+html .fb-like{display:inline}/* ie7 inline block fix*/
```

Comment:
Use `//` comments everywhere<br>Colon after property should be followed by one space<br>Opening curly brace `{` should be preceded by one space<br>Declaration should be terminated by a semicolon